install_files/ansible-base/roles/app/templates/sites-available/focal/journalist.conf

```diff
@@ -0,0 +1,80 @@
+ServerName {{ securedrop_app_apache_listening_address }}
+<VirtualHost {{ securedrop_app_apache_listening_address }}:8080>
+
+# WSGI settings for Flask app for Source Interface
+WSGIDaemonProcess journalist processes=2 threads=30 display-name=%{GROUP} python-path=/var/www/securedrop
+WSGIScriptAlias / /var/www/journalist.wsgi process-group=journalist application-group=journalist
+WSGIPassAuthorization On
+
+# Tell the browser not to cache HTML responses in order to minimize the chance
+# of the inadvertent release or retention of sensitive data. For more, see
+# https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.9.2.
+Header set Cache-Control "no-store"
+
+# Configure X-Sendfile for more efficient large file downloads.
+# Modern versions of WSGI wrapper may make this obsolete, more
+# research required.
+XSendFile        On
+XSendFilePath    /var/lib/securedrop/store/
+XSendFilePath    /var/lib/securedrop/tmp/
+
+Header edit Set-Cookie ^(.*)$ $1;HttpOnly
+Header always append X-Frame-Options: DENY
+Header set Referrer-Policy "no-referrer"
+Header set X-XSS-Protection: "1; mode=block"
+Header set X-Content-Type-Options: nosniff
+Header set X-Download-Options: noopen
+Header set Content-Security-Policy: "default-src 'none'; script-src 'self'; style-src 'self'; img-src 'self'; font-src 'self';"
+
+# Limit the max submitted size of requests.
+LimitRequestBody 524288000
+
+# Set default to deny all access from all filepaths.
+<Directory />
+  Options None
+  AllowOverride None
+  Require all denied
+</Directory>
+
+# Permit limited access specifically to the SecureDrop wsgi files.
+<Directory /var/www/>
+  Options None
+  AllowOverride None
+  <Limit GET POST HEAD>
+    Require ip {{ securedrop_app_apache_allow_from }}
```

Comment:
I'm not sure if a  `Require all denied` is needed here too. In the absence of RequireAny/RequierAll tags, it will default to RequireAny[1] which should be the behavior that we want 

[1] https://httpd.apache.org/docs/2.4/upgrading.html

Reply:
it's just weird to not see a deny-by-default rule here

Reply:
Relevant docs here:

* https://httpd.apache.org/docs/2.4/howto/access.html
* https://httpd.apache.org/docs/2.4/mod/mod_authz_core.html#require

Worth being explicit about the default states, will add.